src/barplot.ts

```diff
@@ -69,41 +78,61 @@ export class BarPlot extends AVisInstance implements IVisInstance {
     return <Element>this.$node.node();
   }
 
+  transform(scale?: [number, number], rotate: number = 0) {
+    const bak = {
+      scale: this.options.scale || [1, 1],
```

Comment:
So it should be like this: `scale: [this.options.scale.slice(0), this.options.scale.slice(1)] || [1, 1],` ?
The backup was done like that in other vises. Shall I correct it everywhere?